src/injected/styles/injected.scss

```diff
@@ -65,14 +65,11 @@
         max-width: 25%;
         user-select: text;
         min-width: 520px;
-        svg {
-            @media screen and (forced-colors: active) {
-                fill: inherit !important;
-            }
-        }
-        path {
+        svg path {
             @media screen and (forced-colors: active) {
-                fill: inherit !important;
+                forced-color-adjust: none !important;
+                stroke: ButtonText !important;
+                fill: ButtonText !important;
```

Comment:
Setting the fill to `ButtonText` is intentional here; otherwise the close button is just an outline